Added a few more options and ordered users by date of last post

keithws opened the pull request:
I really like this plugin, so I added a few features.

* Order contributors by date of last post in reverse-cronological order
* Add option to show/hide the contributor’s avatar
* Add option to show/hide the contributor’s name
* Add option to blacklist users from the recent contributors list by user id
* Update call to get users to filter out subscribers and limit the fields returned to only those needed

Let me know if you'd like to pull this in or if you'd like me to change some stuff.

keithws committed:
order users by date of last post; option to show/hide avatar

theukedge commented:
-- Please reply above this line --

** WE ARE CURRENTLY OUT OF THE OFFICE FOR THE 4TH OF JULY. WE WILL
RETURN TO THE OFFICE ON JULY 5TH. **

Hello Theukedge/recent-Contributors-Widget,

We've received your email and will respond as soon as we can.

We usually respond within one business day and our normal business
hours are Monday-Friday, 7am-4pm Pacific time.

If this is an urgent matter (your website is down, or you need to
restore from a backup), you can respond with the word "urgent", and
I'll review your message as soon as possible and determine the best
course of action.

Thanks

keithws committed:
restored post count

keithws committed:
added title to links

keithws commented:
I noticed a bug showing all users' post counts as zero and I fixed it. I also added title attribute to better describe the links (based on link destination.)

theukedge commented:
-- Please reply above this line --

Hello Theukedge/recent-Contributors-Widget,

We've received your email and will respond as soon as we can.

We usually respond within one business day and our normal business
hours are Monday-Friday, 7am-4pm Pacific time.

If this is an urgent matter (your website is down, or you need to
restore from a backup), you can respond with the word "urgent", and
I'll review your message as soon as possible and determine the best
course of action.

Thanks

theukedge commented:
@keithws Keith, thanks for your contribution. It'll take me a little while to get to this, but I definitely plan to merge this in. Cheers for taking the time to do this.

keithws commented:
Thank you for the response. I tried to follow your coding style and I tried to not to change too much. You'll see in the diff that it's mostly adds for the new features, but I did end up with many changes to the block of code that actually renders the HTML to display the widget. I like what I ended up with, but I'm open to debate on it.
